test-scripts/axe-e2e/README.md

```diff
@@ -0,0 +1,16 @@
+# axe-e2e
+A simple script to analyze multiple pages for accessibility errors, and output
+them to a terminal window, or CSV file. The script runs automatically, and is
+designed to evaluate large numbers of pages.
+
+## Usage
+1. Add the `/axe-e2e/` directory to your project under test
```

Comment:
Could you clarify this, I'm not sure what `under test` means here.

Reply:
Yeah, that could have been worded better. Just pushed an update.